edb/schema/links.py

```diff
@@ -288,6 +288,17 @@ def validate_object(
                 context=srcctx,
             )
 
+        if (
+            self.get_attribute_value('required') and
```

Comment:
Check the values on `scls` with `get_required`/etc. That will ensure it works in all cases, regardless of what things are specified in the actual delta